.happy/terraform/modules/sfn_config/main.tf

```diff
@@ -8,8 +8,8 @@ locals {
       )
     }
     OutputPrefix = "s3://${var.swipe_comms_bucket}/swipe${var.remote_dev_prefix}/${var.app_name}/results",
-    RunSPOTMemory = var.memory,
-    RunEC2Memory = var.memory,
+    RunSPOTMemory = var.memory - 2000,
```

Comment:
The explanation in the PR description is great to explain this, but the `- 2000` here sure feels like a magic number. Could we add a comment here or somewhere? I almost never go looking for PRs and reading PR convos on GitHub so I don't think I would remember why this is here in a month.